types/ember__test-helpers/v1/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "baseUrl": "../../",
+        "typeRoots": [
+            "../../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "paths": {
+            "@ember/*": ["ember__*"],
+            "@ember/test-helpers": ["ember__test-helpers/v1"],
+            "@ember/test-helpers/*": ["ember__test-helpers/v1/*"]
```

Comment:
It seems like CI wants this key to be mangled... but I don’t know why that wasn’t the case for this one: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/ckeditor__ckeditor5-utils/v10/tsconfig.json#L15

Seems like we still have a bug... I’ll look into it today.